web-api/openapi/front_office.yaml

```diff
@@ -4,29 +4,41 @@ info:
   version: 1.0.0-rc6
   description: |-
     # Front Office
-    The system identifies a user with only an id.
-    There are two main endpoints: one that allows registering an interaction between a user and a document (a user clicked or liked a document), and one to retrieve a personalized set of documents for the user. The system needs a few user interactions with documents before it can generate a personalized list.
+    The front office is typically used within front-end apps, for example a website or a mobile application.
+    With this SDK, you can handle interactions with documents and request a list of personalized documents.
 
-    # Auth
-    To authenticate with the API, a token will be provided.
+    ## User
+    Each method requires a `user_id`.
+    From Xayn's perspective, a `user_id` is simply required to group interactions together. We don't need to know who that user is, so it is preferred to create this `user_id` in a privacy-preserving way.
+    For example, create a hash method which transforms your user into an ID hash.
+
+    ## Document
+    By document, we refer to a cohesive text, for example a complete news article.
+    However, we only require a simple representation for our system to work. Consisting just of a unique ID, a text snippet and optional properties.\
+    The text snippet is ideally a short, meaningful representation of the larger document, reduced to just one or two paragraphs.
```

Comment:
This api does not know about the snippet. A document here has an id, some properties, and a score associated to it.